src/utils/platform.ts

```diff
@@ -0,0 +1,21 @@
+// https://stackoverflow.com/questions/9038625/detect-if-device-is-ios
+
+import { Capacitor } from "@capacitor/core";
+
+export function iosNotNative() {
+    if (Capacitor.getPlatform() === "ios") {
```

Comment:
should make this `Capacitor.isNativePlatform() && Capacitor.getPlatform() === "ios"`